src/plugin/message.cc

```diff
@@ -69,6 +69,9 @@ namespace chapel {
       "}\n"
       "\n");
 
+    printer->Print("var unknowFieldStream: bytes = \"\";\n");
```

Comment:
Looks like you've got a typo here: missing the last "n" in "unknown"

Reply:
It happens at other parts of the generation as well, as a heads up

Reply:
Ahhh :sweat_smile: , thanks for pointing out I will fix it!